CHANGES.md

```diff
@@ -1,5 +1,36 @@
 # Changes
 
+## edge-21.11.2
+
+This edge release introduces a new Services workload in the web dashboard that
+shows live calls and route metrics for meshed services. Additionally, the
+`proxy-init` container is no longer enforced to run as root.
+
+The proxy can now retry requests with a `content-length` header—permitting
+requests emitted by grpc-go to be retried. It can also be built using a
+`boringssl` backend instead of the default `rustls` backend.
+
+* Removed hardcoding that enforced the `proxy-init` container to run as root
+* Added support for retrying requests without a `content-length` header
+* Changed service discovery logs from `TRACE` to `DEBUG`
+* Fixed issue with policy controller where it assumed `linkerd` was the name of
+  the control plane namespace, leading to issues with installations that use a
+  non-default namespace name
+* Added support for ephemeral storage requests and limits configured either
+  through the CLI or annotations (thanks @michaellzc!)
+* Deprecated support for topology keys and added support for topology aware
+  hints
+* Added `logFormat` and `logLevel` configuration values for the `proxy-init`
+  container (thanks @gusfcarvalho!)
+* Added services to the web dashboard (thanks @krzysztofdrys!)
+* Updated commands in the web dashboard to use the `viz` subcommand when
+  necessary (thanks @mikutas!)
+* Added support for building the proxy using a `boringssl` backend instead of
+  the default `rustls` backend
+* Fixed identity overrides for endpoint slices
```

Comment:
we can omit this, I think.  this was basically only used by mutlicluster+endpoint slices but that feature still doesn't work end to end yet.